app/policies/user_policy.rb

```diff
@@ -0,0 +1,27 @@
+class UserPolicy < ApplicationPolicy
+  attr_reader :current_user, :user
+
+  def initialize(current_user, user)
+    @current_user = current_user
+    @user = user
+  end
+
+  def show?
+    current_user == @user ||
```

Comment:
Not quite. Double check that your photos feeds are displaying properly. This logic should live somewhere else in your policy. Perhaps, a new policy method?